src/main/java/games/strategy/triplea/ai/proAI/util/ProMatches.java

```diff
@@ -290,7 +296,8 @@
   }
 
   public static Predicate<Territory> territoryIsEnemyNotNeutralOrAllied(final PlayerID player, final GameData data) {
-    final Predicate<Territory> alliedLand = Match.allOf(Matches.territoryIsLand(), Matches.isTerritoryAllied(player, data));
+    final Predicate<Territory> alliedLand =
+        Match.allOf(Matches.territoryIsLand(), Matches.isTerritoryAllied(player, data));
     return Match.anyOf(territoryIsEnemyNotNeutralLand(player, data), alliedLand);
   }
```

Comment:
This method now requires a Javadoc because it has more than two lines due to wrapping.  You can probably just inline `alliedLand` or extract a new private method:

```java
  public static Predicate<Territory> territoryIsEnemyNotNeutralOrAllied(final PlayerID player, final GameData data) {
    return Match.anyOf(territoryIsEnemyNotNeutralLand(player, data), territoryIsAlliedLand(player, data));
  }

  private static Predicate<Territory> territoryIsAlliedLand(final PlayerID player, final GameData data) {
    return Match.allOf(Matches.territoryIsLand(), Matches.isTerritoryAllied(player, data));
  }
```